### PR TITLE
Feature/dfe 592 implement hotjar

### PR DESCRIPTION
**Pull request by shakes90**

### CODE REVIEW

File: src/explore-education-statistics-frontend/.env [hunk 1]
```diff
@@ -3,3 +3,5 @@ DATA_API_BASE_URL=http://localhost:5000
 FUNCTION_API_BASE_URL=http://localhost:7071
 GA_TRACKING_ID=UA-xxxxxxxxx-1
 GA_TRACKING=false
+HOTJAR_TRACKING=true
```
Comment (jamescutts) [inline src/explore-education-statistics-frontend/.env]: Ideally we only want this enabled on live so as to not skew any stats with our local development / testing habbits. But happy to merge this in to allow us to test it provided we revert this before marking as complete

File: src/explore-education-statistics-frontend/.env.dev01 [hunk 2]
```diff
@@ -3,3 +3,5 @@ DATA_API_BASE_URL=https://s101d01-as-ees-data.azurewebsites.net
 FUNCTION_API_BASE_URL=https://s101d01-fa-ees-notify.azurewebsites.net
 GA_TRACKING_ID=UA-139424173-2
 GA_TRACKING=true
+HOTJAR_TRACKING=true
```
Comment (jamescutts) [inline src/explore-education-statistics-frontend/.env.dev01]: Ideally we only want this enabled on live so as to not skew any stats with our local development / testing habbits. But happy to merge this in to allow us to test it provided we revert this before marking as complete

File: src/explore-education-statistics-frontend/.env.dev02 [hunk 3]
```diff
@@ -3,3 +3,5 @@ DATA_API_BASE_URL=https://s101d01-as-eesdev02-data.azurewebsites.net
 FUNCTION_API_BASE_URL=https://s101d01-fa-eesdev02-notify.azurewebsites.net
 GA_TRACKING_ID=UA-139424173-3
 GA_TRACKING=true
+HOTJAR_TRACKING=true
```
Comment (jamescutts) [inline src/explore-education-statistics-frontend/.env.dev02]: Ideally we only want this enabled on live so as to not skew any stats with our local development / testing habbits. But happy to merge this in to allow us to test it provided we revert this before marking as complete

File: src/explore-education-statistics-frontend/.env.dev03 [hunk 4]
```diff
@@ -3,3 +3,4 @@ DATA_API_BASE_URL=https://s101d01-as-eesdev03-data.azurewebsites.net
 FUNCTION_API_BASE_URL=https://s101d01-fa-eesdev03-notify.azurewebsites.net
 GA_TRACKING_ID=UA-139424173-4
 GA_TRACKING=true
+HOTJAR_TRACKING=false
```
Comment (jamescutts) [inline src/explore-education-statistics-frontend/.env.dev03]: We want enabled on live.

File: src/explore-education-statistics-frontend/.env.dev01 (hunk 2 above)
Comment (shakes90) [reply src/explore-education-statistics-frontend/.env.dev01]: Yeah just for testing purposes in the first instance.

File: src/explore-education-statistics-frontend/.env (hunk 1 above)
Comment (shakes90) [reply src/explore-education-statistics-frontend/.env]: Yep merge and I'll test but yeah not going to enable for prod.

**Pull request merged**